Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
It's used for local testing and validating the UI works manually (as
mentioned in the PR description)

On Tue, Nov 10, 2015 at 3:06 PM, George Schneeloch <notifications@github.com
> wrote:

> In Gemfile <https://github.com/mitodl/ccxcon/pull/7#discussion_r44457518>:
>
> > @@ -0,0 +1,3 @@
> > +source 'https://rubygems.org'
>
> Where do we use the apiaryio gem here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mitodl/ccxcon/pull/7/files#r44457518>.
>